app/models/manageiq/providers/ansible_tower/shared/inventory/parser/automation_manager.rb

```diff
@@ -45,10 +45,11 @@ def configuration_scripts
 
   def configuration_script_sources
     collector.projects.each do |project|
+
       inventory_object = persister.configuration_script_sources.find_or_build(project.id.to_s)
       inventory_object.description = project.description
       inventory_object.name = project.name
-      inventory_object.authentication = persister.credentials.lazy_find(project.credential_id.to_s)
+      inventory_object.authentication = persister.credentials.lazy_find(project.credential_id.to_s) if project.credential
```

Comment:
@Ladas what would happen if this would resolve to `lazy_find('')`? I.e. is `lazy_find` smart enough not to crash if passed unresolvable stuff?